modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -201,7 +335,7 @@ BINLOG_FORMAT = STATEMENT and at least one table uses a storage engine limited t
 
 Please refer to xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-with-binary-logging-enabled[MySQL / MariaDB with Binary Logging Enabled] on how to correctly configure your environment.
 
-Occasionally, _files do not show up after an upgrade_. A rescan of the files can help:
+In the unlikely case, that files do not show up in the web-ui after the upgrade, do a files scan to have them visible again.
```

Comment:
I would either link to the occ page with he command or write the command here. like occ files:scan --all